docs/release-notes.md

```diff
@@ -1,3 +1,13 @@
+## 0.14.1
+
+November 9th, 2020
+
+### Removed
+
+* `UJSONResponse` was removed (this change was intended to be included in 0.14.0). Please see the
```

Comment:
The additional hint is nice; could we please still add the PR number, for the sake of traceability? (This is what I meant in my last suggestion adding `Pull #N`. I find they make it easier to find back the set of changes and provide an additional way to evaluate what actually changed.)

Reply:
Agree. Was going for brevity/similarity to previous changelog entries, but will rather go for (succinct) detail in the future 👍.